src/components/MessageCenter/MessageCenterTrigger.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PfReact from 'patternfly-react';
+
+type StateType = {};
+type PropsType = {
+  newMessagesCount: number;
+  toggleMessageCenter: () => void;
+};
+
+export default class MessageCenterTrigger extends React.PureComponent<PropsType, StateType> {
```

Comment:
I personally prefer for stateless components:
`const PfHeader: React.SFC<PfHeaderProps> = props => {
  return <div className="page-header">{props.children}</div>;
};
export default PfHeader`
That way you aren't even allowed to add state. Which most all of the state will be in Redux now anyway.

Reply:
I didn't know what was our stand for using functional components.
I can change that if is OK to mix then both (functional components vs class components)